src/Sdk/WebApi/WebApi/Results/Contracts.cs

```diff
@@ -0,0 +1,36 @@
+namespace GitHub.Services.Results.Contracts
+{
+    public class GetSignedStepSummaryURLRequest
```

Comment:
are these `DataContract`?
if so, you might want to follow the C# conversion in other file and adding `[DataContract]` and `[DataMember]` to all the class/property.

Reply:
Also, you need to name it `WorkflowJobRunBackendId` for styling.

Reply:
Those are Twirp service requests.  However I think they are just json anyway.  I will add the contract attributes.  

It's `WorkflowJobRun*`. 🤷🏼